WordPress/Classes/ViewRelated/Pages/PageListTableViewHandler.swift

```diff
@@ -9,11 +9,13 @@ final class PageListTableViewHandler: WPTableViewHandler {
     }
 
     private var pages: [Page] = []
-
+    private let blog: Blog
     private lazy var publishedResultController: NSFetchedResultsController<NSFetchRequestResult> = {
         let publishedFilter = PostListFilter.publishedFilter()
         let fetchRequest = NSFetchRequest<NSFetchRequestResult>(entityName: Page.entityName())
-        fetchRequest.predicate = publishedFilter.predicateForFetchRequest
+        let predicate = NSPredicate(format: "blog = %@ && revision = nil", blog)
```

Comment:
One tiny nitpick: Can we make this use `#keyPath()` instead of using stringly-typed property names? Otherwise, good to go :)